warehouse/upload.go

```diff
@@ -1595,7 +1594,23 @@ func (job *UploadJobT) Aborted(attempts int, startTime time.Time) bool {
 }
 
 func (job *UploadJobT) setUploadError(statusError error, state string) (string, error) {
-	pkgLogger.Errorf("[WH]: Failed during %s stage: %v\n", state, statusError.Error())
+	defer func() {
+		pkgLogger.Errorw("upload error",
+			logfield.JobID, job.upload.ID,
```

Comment:
Should this be a warning? Given the impact and the frequency of upload errors.

I am cornered that jobID can be confused with the job_id of sources, should we name it upload_job_id to make it unambiguous? 

```suggestion
		pkgLogger.Errorw("upload error",
			logfield.UploadJobID, job.upload.ID,
```